package.json

```diff
@@ -57,29 +57,29 @@
   "devDependencies": {
     "@angular/cli": "^1.2.1",
     "@angular/compiler-cli": "^4.3.0",
-    "@types/chart.js": "^0.0.12",
+    "@types/chart.js": "^2.6.8",
     "@types/chroma-js": "^1.3.0",
     "@types/emojione": "^2.2.2",
-    "@types/jasmine": "~2.5.53",
+    "@types/jasmine": "~2.6.0",
     "@types/jstimezonedetect": "^1.0.0",
-    "@types/moment-timezone": "^0.2.34",
-    "@types/node": "^6.0.83",
+    "@types/moment-timezone": "^0.5.0",
+    "@types/node": "^8.0.34",
     "@types/scriptjs": "^0.0.2",
     "@types/socket.io-client": "^1.4.30",
-    "codelyzer": "~3.0.1",
-    "jasmine-core": "~2.5.2",
-    "jasmine-spec-reporter": "~4.1.1",
+    "codelyzer": "~3.2.1",
+    "jasmine-core": "~2.8.0",
+    "jasmine-spec-reporter": "~4.2.1",
     "karma": "~1.7.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-cli": "^1.0.1",
     "karma-coverage-istanbul-reporter": "^1.3.0",
     "karma-jasmine": "^1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
-    "ng2-mock-component": "^0.0.6",
+    "ng2-mock-component": "^0.1.0",
     "protractor": "^5.1.2",
     "sw-precache": "^5.2.0",
-    "ts-node": "~3.2.0",
+    "ts-node": "~3.3.0",
     "tslint": "^5.5.0",
-    "typescript": "~2.4.1"
+    "typescript": "^2.3.4"
```

Comment:
This was downgraded because of angular cli warning. Valid?

Reply:
yep

Reply:
https://github.com/angular/angular-cli/issues/7625